src/components/comment-list-item.component.js

```diff
@@ -284,29 +321,43 @@ class CommentListItemComponent extends Component {
                 {comment.user.login}
                 {'  '}
               </Text>
-            </TouchableOpacity>}
+            </TouchableOpacity>
+          )}
 
           <View style={styles.dateContainer}>
             <Text style={styles.date}>
               {moment(comment.created_at).fromNow()}
             </Text>
           </View>
+
+          {isActionButtonVisible && (
+            <View style={styles.iconContainer}>
```

Comment:
Let's for greater clarity, rename the class to something like `actionButtonIconContainer`?